tree-sitter-stack-graphs/src/loader.rs

```diff
@@ -325,8 +335,15 @@ impl Loader {
         graph: &mut StackGraph,
         cancellation_flag: &dyn CancellationFlag,
     ) -> Result<(), LoadError<'a>> {
-        let file = graph.add_file(&path.to_string_lossy()).unwrap();
+        let file_name = path.to_string_lossy();
+        let file: stack_graphs::arena::Handle<stack_graphs::graph::File> =
+            graph.add_file(&file_name).unwrap();
         let mut globals = Variables::new();
+
+        globals
+            .add(FILE_PATH_VAR.into(), path.to_str().unwrap().into())
+            .expect("failed to add file path variable");
```

Comment:
The builtins are a bit special and the actual path is not relevant. They are usually definitions that are implicitly defined, like a runtime library or something.

I'm not sure why we are using `path` here in `add_file`. I think we should be using `"<builtins>"` here as well, both in `add_file` and as the value in `globals`. As a little code cleanup we could define a top-level constant `BUILTINS_FILENAME` and replace the literals.

Reply:
I wonder if having chevrons could cause issues (as it's not a standard path character I believe). But it was working as is so let's keep it